compiler-core/src/format.rs

```diff
@@ -920,11 +920,15 @@ impl<'comments> Formatter<'comments> {
                 // If there's more than one item in the tuple and there's a
                 // pipeline or long binary chain, we want to indent those to
                 // make it easier to tell where one item ends and the other
-                // starts.
-                if elements.len() > 1 && (e.is_binop() || e.is_pipeline()) {
+                // starts, except if they are preceded by a comment.
+                let mut comments = self_.pop_comments(e.location().start).peekable();
+                if comments.peek().is_none()
+                    && elements.len() > 1
+                    && (e.is_binop() || e.is_pipeline())
```

Comment:
I'm not sure this is the correct behaviour, we still want things to get nested even if there's a comment. How would this bit of code be indented with this change?
```gleam
wibble(
  // comment
  wobble
    |> fun
    |> fun2,
  wabble,
)
```
If I understand this correctly, with this change, the pipe wouldn't be nested

Reply:
Gotcha. I added something equivalent to your code sample as a test case to ensure that we still nest properly.